release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -3190,6 +3190,8 @@ There is currently no workaround to resolve this issue if encountered. To avoid
 
 * If a cluster that was deployed through ZTP has policies that do not become compliant, and no `ClusterGroupUpdates` object is present, you must restart the {cgu-operator} pods. Restarting {cgu-operator} creates the proper `ClusterGroupUpdates` object, which enforces the policy compliance. (link:https://issues.redhat.com/browse/OCPBUGS-4065[*OCPBUGS-4065*])
 
+* Currently, a certificate compliance issue, specifically outputted as `x509: certificate is not standards compliant`, exists when you run the `openshift-installer` on macOS for the purposes of installing an {product-title} cluster on VMware vSphere. This issue relates to a known issue with the `golang` compiler in that the compiler does not recognize newly supported macOS certificate standards. No workaround exists for this issue. (link:https://issues.redhat.com/browse/OSDOCS-5694[*OSDOCS-5694*])
```

Comment:
Thanks for the suggestion. I might leave the text as is because I have three PRs created, and I want to maintain consistency. I appreciate the suggested edit!